index.js

```diff
@@ -9,18 +12,29 @@ const EDITOR_MODULE = {
   worker: {
     id: 'vs/editor/editor',
     entry: 'vs/editor/editor.worker',
-    output: 'editor.worker.js',
```

Comment:
This is no longer required as the `[name]` attribute used to resolve the filename template is based on the entry filename. All generated files have the same names as before unless `filename` is specified.